### PR TITLE
TF-3789:  Add launch_manager support

### PR DESCRIPTION
**Pull request by PGlaubitzWork**

### CODE REVIEW

File: robotiq_force_torque_sensor/package.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<package>
+<package  format="2">
```
Comment (adamcalhoon) [inline robotiq_force_torque_sensor/package.xml:2]:
```suggestion
<package format="2">
```

File: robotiq_force_torque_sensor/package.xml [hunk 2]
```diff
@@ -11,15 +11,13 @@
...
-  <run_depend>roscpp</run_depend>
-  <run_depend>std_msgs</run_depend>
-  <run_depend>message_runtime</run_depend>
-
+  <depend>bondcpp</depend>
+  <depend>launch_manager_interface</depend>
+  <depend>roscpp</depend>
+  <depend>std_msgs</depend>
+  <exec_depend>message_runtime</exec_depend>
+
```
Comment (adamcalhoon) [inline robotiq_force_torque_sensor/package.xml:20]:
```suggestion

```

**Review by adamcalhoon — COMMENTED** [review]
Do you need to add a launch_status_topic arg to the launch file(s)?

**Review by adamrr — APPROVED** [review]

**Review by mechterp10 — APPROVED** [review]

**Comment by mechterp10** [comment]
Feel free to open up your refactored PR next week for 2.3.1 or 2.4.0

**Pull request merged**